index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+    href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+    rel="stylesheet"
+    integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+    crossorigin="anonymous"
+  />
+  <link
+    href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+    rel="stylesheet"
+  />
+  <link href="./styles.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Domine&family=Inter:wght@400;700&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Montserrat:wght@100&display=swap" rel="stylesheet"/> 
+    <title>Home Page</title>
+</head>
+<body onload="showFeatures()">
+ 
+   
+        <section class="page-one" >
+            <nav id = 'navbar'>
+            <div class="banner desktop"> 
+                <nav><a href="#"><img src="./images/About/facebook.svg" alt="facebook-logo"></a></nav>
+                  <nav><a href="#"><img src="./images/About/twitter.svg" alt="twitter-logo"></a></nav>
+                  <nav><a href="#">English</a></nav>
+                  <nav><a href="#">My page</a></nav>
+                  <nav><a href="#">Logout</a></nav>
+            </div>
+            <div class="toolbar">
+                <a id="hamburger" class=" hamburger" href="#">
+                    <img onclick="popmenu()" alt="Hamburger Icon" src="./images/hamburger.svg"></a>
+                <a class = "logo" href="#"><img alt="Logo Icon" src="./images/logo.svg"></a>
+                <div class="toolbar2 show" id="visible" >
+                    <div>
+                        <img onclick ="popmenu()" src="./images/Home/Icon_Cancel.png" alt="closebtn">
+                    </div>
+                    <a class ="toollink" href="./about.html">About</a>
+                    <a class ="toollink" href="#">Program</a>
+                    <a class ="toollink" href="#">Join</a>
+                    <a class ="toollink" href="#">Sponsor</a>
+                    <a class ="toollink" href="#">News</a>
```

Comment:
- [ ] [OPTIONAL] In addition, I have noticed that your toolbar is crossing over the toolbar which affects the responsiveness of the application. Kindly ensure to make the width of the toolbar responsive in relation to the links to make the application more professional ☺️.
- [ ] [OPTIONAL] Finally, please try as much as possible to follow the guidelines of the original design. You may need to align the logo in the center to match the original design of the application ☺️.
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/235321611-033b5e84-41b6-4e22-bfae-6be2fd526f6a.png)